src/vivarium/framework/population/population_view.py

```diff
@@ -130,7 +142,9 @@ def get_private_columns(
         query_columns
             The (public) column(s) needed to evaluate the query string.
         query
-            Additional conditions used to filter the index.
+            Additional conditions used to filter the index. Note that it will
+            *not* be combined with this PopulationView's query property (in order
```

Comment:
I think it may be confusing to users how sometimes the pop-view's query is included and sometimes not. We should talk about whether this should be handled differently.